modules/ROOT/pages/mobile-cli.adoc

```diff
@@ -0,0 +1,109 @@
+
```

Comment:
I can view this page in https://deploy-preview-811--aerogear-docs-preview.netlify.com/aerogear/latest/mobile-cli/ but there does not appear to be a way of navigating through the site to get to it.  Should there be an option in the menu sidebar to click on?